shared/common-adapters/index.js.flow

```diff
@@ -33,3 +33,43 @@ export {default as UserBio} from './user-bio'
 export {default as UserCard} from './user-card'
 export {default as UserProofs} from './user-proofs'
 export {default as Usernames} from './usernames'
+
+const NativeActivityIndicator: Object = {}
+const NativeAnimated: Object = {}
+const NativeClipboard: Object = {}
+const NativeEasing: Object = {}
+const NativeImage: Object = {}
+const NativeLinking: Object = {}
+const NativeListView: Object = {}
+const NativeModal: Object = {}
+const NativeNavigator: Object = {}
+const NativePicker: Object = {}
+const NativeScrollView: Object = {}
+const NativeStyleSheet: Object = {}
+const NativeSwitch: Object = {}
+const NativeText: Object = {}
+const NativeTextInput: Object = {}
+const NativeTouchableHighlight: Object = {}
+const NativeTouchableOpacity: Object = {}
+const NativeTouchableWithoutFeedback: Object = {}
+
+export {
```

Comment:
i killed the index.js.flow entirely and now there's just index.js (shared) and index.native.js shared + native stuff. i think this does what you want